plugins/module_utils/_api_data.py

```diff
@@ -3942,6 +3942,30 @@ def join_path(path):
             },
         ),
     ),
+    ('routing', 'igmp-proxy'): APIData(
+        unversioned=VersionedAPIData(
+            single_value=True,
+            fully_understood=True,
+            fields={
+                'query-interval': KeyInfo(),
+                'query-response-interval': KeyInfo(),
+                'quick-leave': KeyInfo(default=False),
+            },
+        ),
+    ),
+    ('routing', 'igmp-proxy', 'interface'): APIData(
+        unversioned=VersionedAPIData(
+            fully_understood=True,
+            fields={
+                'alternative-subnets': KeyInfo(),
+                'comment': KeyInfo(can_disable=True, remove_value=''),
+                'disabled': KeyInfo(default=False),
+                'interface': KeyInfo(),
```

Comment:
Shouldn't `interface` be the primary key here? Or can you have multiple entries per interface?

Reply:
Yes, you're right. Done.